src/custommem.c

```diff
@@ -2085,18 +2077,23 @@ void relockCustommemMutex(int locks)
 
 static void init_mutexes(void)
 {
-    #ifdef USE_CUSTOM_MUTEX
+#ifdef _WIN32 // TODO: better wow64 support?
+    InitializeCriticalSection(&mutex_blocks);
+    InitializeCriticalSection(&mutex_prot);
+#else
+#ifdef USE_CUSTOM_MUTEX
```

Comment:
shouldn't USE_CUSTOM_MUTEX be the top test and then WIN32/LINUX in the fallback?